ChangeLog

```diff
@@ -9,10 +9,18 @@ Release date: TBA
 ..
   Put new features here and also in 'doc/whatsnew/2.14.rst'
 
+* Added new checker ``comparison-of-constants``
+
+  Closes #6076
+
 * Improved wording of the message of ``deprecated-module``
 
   Closes #6169
 
+* added new lint ``incorrect-number-of-parameters`, which checks most built-ins (tuple, oct and the like)
```

Comment:
I'm not sure where this is coming from. I feel like your strategy of `git merge` and `git rebase` might not like how we handle branches. Are you sure the `remotes` are set up correctly? I'd be happy to help fix it!

Reply:
> Are you sure the remotes are set up correctly?

I have `origin  git@github.com:omarandlorraine/pylint.git (fetch)` and the same for `(push)`. So what I'm currently doing is to merge PyCQA main into omarandlorraine's main, and then rebasing logic_checks on that. This message comes from some work that I did over on [another pull request](https://github.com/PyCQA/pylint/pull/6280) which somehow ended up in my main branch. I'll see if I can maybe revert that commit. 

But maybe there's an easier way than the merge/rebase? How are branches normally handled on this project?

Reply:
What I generally do is create new branches of off `upstream/main`. That way there is no need to keep `origin/main` up to date and you remove one step in the process.

Then, whenever a merge conflict arises I just `git pull upstream main` and resolve the conflict and `git push origin branch`. If I want to clean up the commit history and remove any merge conflicts I use `git pull --rebase upstream main`, but note that this is generally not recommended within this project as you will then need to force push your changes. This makes it difficult for reviewers to keep track of what they have already reviewed.